### ISSUE TITLE
File checksum error when trying to upload image to DTR

### ISSUE THREAD

**Issue by doverholt** [docker/for-mac#2497]
### Expected behavior

A successful upload of the docker image to DTR

### Actual behavior

Repeated retries for 2 layers of the Docker image. The two layers that repeatedly fail are 118.8 MB and 378.1 MB in size.

A file integrity checksum failed for "opt/tomcat/webapps/webapp.war" error message.

### Information

Docker for Mac 17.12.0-ce-mac49 (21995)
Diagnostic ID: 50643BD6-B431-426F-8BB0-7AE1FC60A464

### Steps to reproduce the behavior

Docker push of the image
Get file integrity error

I tried this on two separate DTR instances, both failed with the same error.

**Comment by richardasaurus** [comment]
I'm getting this when doing `docker push` to aws ecr

```
file integrity checksum failed for \"usr/lib/x86_64-linux-gnu/libm.a
```

mac os high sierra, Docker Version 17.12.0-ce-mac49 (21995).

Tried reinstalling docker, error persists

**Comment by djs55** [comment]
@doverholt I suspect this was caused by an APFS disk corruption problem #2625 which affected the use of sparse files. The bug has been fixed in recent versions of macOS: 10.13.4 and later. Unfortunately it sounds like your image is corrupt and you will need to reset to factory defaults :(
